modules/common/nyx-overlay.nix

```diff
@@ -20,7 +20,7 @@ let
         else
           import "${flakes.nixpkgs}" {
             inherit (cfg.flakeNixpkgs) config;
-            localSystem = stdenv.hostPlatform;
+            localSystem = flakes.nixpkgs.legacyPackages."${pkgs.system}".stdenv.hostPlatform;
```

Comment:
I'm going to have to test this change a little because `pkgs.system` does not exist in every scenario.